configure.ac

```
@@ -979,6 +979,8 @@ AS_CASE([$host],
     [arch=arm; model=armv8; system=linux_eabi],
   [armv7*-*-linux-gnueabi],
     [arch=arm; model=armv7; system=linux_eabi],
+  [armv7a-*-linux-androideabi*],
```

Should this more pedantically be:
```suggestion
  [armv7a-*-linux-androideabi-*],
```
or is there something else which can be suffixed?

---

The triplet can be something like `armv7a-unknown-linux-androideabi24` so this patch would break it

---

Gotcha, thanks!